website/source/docs/job-specification/job.html.md

```diff
@@ -87,8 +89,16 @@ job "docs" {
 - `update` <code>([Update][update]: nil)</code> - Specifies the task's update
   strategy. When omitted, rolling updates are disabled.
 
+- `vault_token` `(string: "")` - Specifies the Vault token that proves the
```

Comment:
So this shouldn't really be set in the job. It is just to transfer it. It is better to use the `VAULT_TOKEN="123" nomad run job.nomad` and have it set it rather than checked in to the job file